src/components/ElvaYBPractice2/styles/MainComponent.styled.ts

```diff
@@ -0,0 +1,258 @@
+import styled from 'styled-components'
+import { Slider, Switch } from 'antd'
+
+import BackgroundPic from '../../../assets/ElvaYongbin/BackgroundPic.svg'
+import PatternCirclePic from '../../../assets/ElvaYongbin/images/pattern-circles.svg'
+import SliderIcon from '../../../assets/ElvaYongbin/images/icon-slider.svg'
+
+export const MainComponent = styled.div`
+z-index: 2;
+border-radius: 15px;
+width: max-content;
+height: auto;
+display: flex;
+padding-top: 30px;
+background-color: #ffffff;
+align-items: center;
+justify-content: center;
+flex-direction: column;
+box-shadow: 0px 3px 10px rgb(0 0 0 / 0.2);
+@media only screen and (max-width: 600px) {
+  margin-top: 180px;
+  height: 60vh;
+  width: 55vw;
+
+  
+`
+
+export const ToggleContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  margin-left: 120px;
+
+  width: 80%;
+  @media only screen and (max-width: 600px) {
+    width: 100%;
+    margin-left: 40px;
+    top: -10px;
+  }
+`
+export const SlidercontainerDesktop = styled.div`
+  width: 60%;
+  align-items: center;
+  display: block;
+  flex-direction: row;
+
+  @media only screen and (max-width: 600px) {
+    width: 85%;
+    top: -85px;
```

Comment:
consider removing this also, it should work fine without it

Reply:
or u can remove the width and top since the display is already set to none in mobile view

Reply:
Resolved in commit d2f947f.